src/lib/Containers/Conversation.tsx

```diff
@@ -187,38 +192,23 @@ export default createFragmentContainer(Conversation, {
   `,
 })
 
-export interface RelayProps {
+interface RelayProps {
```

Comment:
This is no longer `export`ed (since everything should have their own interface), and I cleaned it up from some of the stuff that did move into `Messages`.